packages/block-library/src/reset.scss

```diff
@@ -126,4 +126,15 @@
 		cursor: revert;
 		transform: revert;
 	}
+
+	// Safari can have renderint issues with CSS3 transitions.
```

Comment:
What is renderint? And which CSS3 transition is causing this? The block moving one?

Reply:
Sorry, typo, "rendering".

I'm not entirely sure which property is causing the issue, and I was hoping that by starting this PR (since it seems to fix the issue for me) as a good way get the conversation going.

In my testing, the flickering happens mostly on the classic gallery buttons (left/right/remove), while you are scrolling:

![gallery](https://user-images.githubusercontent.com/1204802/116855813-4d2e8480-abfa-11eb-8018-1cf20d7e933b.gif)

But according to the initial ticket, it appears to be happening to other blocks as well.